test/no-continue.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @fileoverview Tests for no-continue rule
+ * @author Simon Hajjar <simon.j.hajjar@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require("solium");
+
+var userConfig = {
+        "rules": {
+                "no-continue": "error"
```

Comment:
you need to change this to `"security/no-continue": "error"`. Simply writing the rule name means Solium only checks the core rules for this rule. `A/B` means `B` rule from `A` plugin.

Reply:
Fixed that, thanks!